vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -1,3 +1,177 @@
+
+See [Build and Publish a Project Bundle](../../tutorials/create/pb/publish-project-bundle.md) for more details.
+
+## ent bundle Command List 
+syntax: `ent bundle [commands] [subcommand] [arguments]` 
+| Commands | Subcommands|  Description
+|:--|:--|:--
+|`build`||Build components (MFE, MS) with a selector
+|`info`||Show status information for the bundle project 
+|`init`||Initialize project folder structure and descriptor
+|`ms`|`add`| Add ms project components		 	
+|	| `init`	|	Init MS with scaffolding
+| |	`rm` |	Remove a MS project component
+|`mfe` |	`add`	 |  	Add a MFE project component
+| |	`init`	|	Init with scaffolding 
+| |	`rm` |	Remove MFE project component 	
+|`run`|| Run bundle components 
+|`pack`||Create distribution artifacts (Docker images)
+|`publish`||Publish Docker images to a Docker registry
+
+
+## Initialization 
+
+| Command |  Description
+|:--|:--
+|`ent bundle init [name]` |Initialize a new project with the default structure and files
+|`ent bundle init [name] --from-hub`| Initialize a bundle from an Entando Hub
+
+#### Command Details
+* `ent bundle init [name] --from-hub`: Utilize an existing bundle from an Entando Hub to jump start your project. The ent bundle CLI will pull the package and rebuild the structure as needed, ready for customization. 
+
+## Build
+ 
+| Command| Descriptions
+|:--|:--
+|`ent bundle build [component-name]` | Build a single component
+|`ent bundle build [mfe-1] [ms-2]...`| Build every the component passed as arguments
+|`ent bundle build --all`|Build all the components in the project
+|`ent bundle build --all-ms`|Build all microservices|
+|`ent bundle build --all-mfe`|Build all the microfrontend|
+
+#### Command Details
+`ent bundle build`: Constructs the project files based on the entando.json provided. It executes according to the type of stack your components are built with. For instance, npm build will be performed for a React MFE. A log file is generated for each component inside the .entando/logs directory for your project.
+
+## Run	
+| Command| Descriptions
+|:--|:--
+|`ent bundle run [component-name]` | Locally run your single component
+|`ent bundle run [mfe-1] [ms-2]...`| Locally run all the components passed as arguments 
+|`ent bundle run --all`| Locally run all the components in the bundle
+|`ent bundle run --all-ms`| Locally run the MSs in the bundle
+|`ent bundle run --all-mfe`| Locally run the MFEs in the bundle
+
+#### Command Details
+`ent bundle run`: Executes processes dependent on the component type and stack. For example, MVN spring:boot will execute for a Sping Boot microservice. All the components in the bundle will run in parallel, with a log file generated for each inside the .entando/logs directory.
+
+## Package
+| Command| Descriptions
+|:--|:--
+|`ent bundle pack`|Generate the bundle artifacts, and the bundle Docker image and MS images 
```

Comment:
I think the point there is a main image for the bundle and then individual microservice images. Only reason to keep Docker is because of the shift from git-based bundle to Docker-based bundle but I think Docker can be dropped here.